TUTORIAL.ipynb

```diff
@@ -138,7 +138,7 @@
     "    row['name'] = row['name'].title()\n",
     "\n",
     "Flow(\n",
-    "      load('data/beatles.csv'),\n",
+    "      load('beatles.csv'),\n",
```

Comment:
It was referring to a path that didn't exist (the path is correct in TUTORIAL.md).